### PR TITLE
Make changes specified in PRESS0-685, PRESS0-980, PRESS0-981

### PR DESCRIPTION
**Pull request by aratidgr8**
1. Changes specified in JIRAs PRESS0-685, PRESS0-980, PRESS0-981
2. Correct translations texts formats in all Ecommerce module files
3. Update .pot, .po and .mo files for all languages

## Proposed changes

**PRESS0-685**

- Update text below the header "Sales and Discounts"
- From -"Create and manage shop deals, sales promotions, and upsell campaigns for your store". To - "Create and manage X-sell, Upsell, sales and promotions through campaigns to boost your sales".
- pt-br translation - "Crie e gerencie vendas cruzadas, vendas adicionais, vendas e promoções através de campanhas para impulsionar suas vendas".

**PRESS0-980**

- Given that a visitor/Customer on the wp-admin BH/ HG / HG latum nav
- when they click on BH-> Store option,
- then "Sales & Promotions" page should display "Sales & Promotions" instead of "Sales & Discounts" in the page.
- For Latum  pt-br translation for string "Sales & Promotions" is "Vendas e Promoções"

**PRESS4-981**

- For BH/ HG/ HG Latum
- Under Marketing tab within the wp-admin page update text "Promote"  to "Promotions"
- For latum  pt-br translation for text "Promotions" is "Promoções"

Screenshot 2024-03-27 at 3 23 18 PM

## Type of Change

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: includes/ECommerce.php [hunk 1]
```diff
@@ -446,7 +446,7 @@ function custom_add_promotion_menu_item( $menu_items ) {
     add_submenu_page(
       'woocommerce-marketing',
       'Promotion product Page',
-      __('Promote','wp_module_ecommerce'),
+      __('Promotions','wp-module-ecommerce'),
```
Comment (manikantakailasa) [inline includes/ECommerce.php:449]: this should be in double quotes.

File: includes/ECommerce.php [hunk 2]
```diff
@@ -463,7 +463,7 @@ function custom_product_general_options() {
     wp_enqueue_style ( 'Create_a_Promotion' , NFD_ECOMMERCE_PLUGIN_URL . 'vendor/newfold-labs/wp-module-ecommerce/includes/Promotions.css' , array (), '1.0' , 'all' );
 		echo '<div class="options_group">
             <p class="form-field custom-button-field">
-						  <a id="Create_a_Promotion" href="'.$redirect_url.'" class="promotion">'. __('Create a Promotion', 'wp_module_ecommerce').'</a>
+						  <a id="Create_a_Promotion" href="'.$redirect_url.'" class="promotion">'. __('Create a Promotion', 'wp-module-ecommerce').'</a>
```
Comment (manikantakailasa) [inline includes/ECommerce.php:466]: this should also be in double quotes.

File: includes/ECommerce.php (hunk 1 above)
Comment (wpscholar) [reply includes/ECommerce.php:449]: @manikantakailasa Our coding standards state that we should use single quotes everywhere unless double quotes are required for string interpolation.

File: src/configs/YITHPlugins.config.js [hunk 3]
```diff
@@ -87,12 +87,12 @@ export const YITHPluginsDefinitions = (props) => ({
...
       text: ({ isActive }) => ({
-        title: __('YITH WooCommerce AJAX Search', 'wp-module-ecommerce'),
+        title: __("YITH WooCommerce AJAX Search", 'wp-module-ecommerce'),
         description: __(
-          'YITH WooCommerce AJAX Search is an effective and predictive real-time search engine on your store so users can quickly find products they want. Speed up your search and boost your sales.',
+          "YITH WooCommerce AJAX Search is an effective and predictive real-time search engine on your store so users can quickly find products they want. Speed up your search and boost your sales.",
           'wp-module-ecommerce'
         ),
-        actionName: isActive ? 'Manage' : 'Enable',
+        actionName: isActive ? __("Manage") : __("Enable"),
```
Comment (wpscholar) [inline src/configs/YITHPlugins.config.js:95]: The translation functions here should have a text domain assigned or they won't get translated. The strings should also use single-quotes.

File: src/components/WonderCart.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ export function WonderCart(props) {
   let showInProgress = isInstalling || wonderCartStatus.data?.isInstalling;
   return (
     <Section.Container>
-      <Section.Header title={__("Sales & Discounts", "wp-module-ecommerce")} />
+      <Section.Header title={__("Sales & Promotions", "wp-module-ecommerce")} />
```
[the remark this answers is not in the document]
Comment (aratidgr8) [reply src/components/WonderCart.js]: Done

**Comment by aratidgr8** [comment]
> @aratidgr8 All the places where the code was updated to use double-quotes instead of single quotes should be reverted. You should be able to run `composer run fix` to auto-update those to match the coding standards.

**Comment by aratidgr8** [comment]
@wpscholar PR comment changes are completed. Please help me with review.

**Review by wpscholar — DISMISSED** [review]

**Pull request merged**